script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -6,75 +6,75 @@ The sound of creaking vehicles.
 Life here was nothing but bright. That was all the baggage I needed to take with me.
 <Page14>
 When I left the Arima household, Keiko-san was the only one to see me off.
 It was my own suggestion that I should be as quiet as possible so as not to wake the rest of the family.
 <Page15>
-"Thank you for everything you've done for me.
-Please give my regards to Father and Miyako."
+'Thank you for everything you've done for me.
+Please give my regards to Father and Miyako.'
 The person who had played the role of my mother for seven years――――had a terribly sad look in her eyes. I never thought I'd see her look like that.
 <Page16>
-"I know it will be hard living in the Tohno mansion, but be strong. You're physically weak, so don't be too reckless."
+'I know it will be hard living in the Tohno mansion, but be strong. You're physically weak, so don't be too reckless.'
 I accepted her words of concern with an open heart.
-The past seven years have been truly peaceful. As long as I didn't cause trouble for the Arimas, I couldn't be happier.
+The past seven years have been truly peaceful. If in those years the Arima's didn't feel troubled by me, I couldn't be happier.
 <Page17>
-"After seven years, I'm as healthy as a man can be.
-Even though it looks like this, my body is rather sturdy, you know."
-"Yes, you are right.
-The Tohnos are very different from other families, but you have never seemed to be afraid of them. Ever since you were a child, you have been so fearless that it amazes me."
+'After seven years, I'm as healthy as a man can be.
+Even though it looks like this, my body is rather sturdy, you know.'
+'Yes, you are right.
+The Tohnos are very different from other families, but you have never seemed to be afraid of them. Ever since you were a child, you have been so fearless that it amazes me.'
 <Page18>
 At her words mingled with a bitter smile, I smiled back.
 I was glad that in Keiko-san's mind, I was still her child.
 <Page19>
-"You're overestimating my abilities.
-...I'm off then. Take care of yourself, Mother."
-"Yes, I will. Take care as well, Shiki."
+'You're overestimating me.
+...I'm off then. Take care of yourself, Mother.'
+'Yes, I will. Take care as well, Shiki.'
 It was typical of Keiko-san not to reply with something like 'Have a good day.'
 She sent me off with tears in the corners of her eyes, having already accepted that I would never return.
 <Page20>
 That's what happened just 40 minutes ago.
-It was a turning point in my entire life.
+It was a turning point in my life.
 It was the beginning of a new life for me, and the end of Tohno Shiki's old life.
 <Page21>
 The scenery outside is changing little by little.
-We must have passed through Yashirogi, a suburb across the river, and entered the urban area of Souya.
+We must have passed through Yashirogi, a suburb across the river, and have entered the urban area of Souya.
```

Comment:
Not sure we need the second `have`, if you elide the clause commas you'd get
> We must have passed through Yashirogi and have entered the urban area of Souya.

which seems a bit stiffer than 
> We must have passed through Yashirogi and entered the urban area of Souya.

Reply:
> 川を隔てて郊外だったやしろぎを抜け、都市部であるそうやに入ったのだろう。

It seems that we passed through Yashirogi, a suburb across the river, before entering the urban area of Souya.
(Don't see a need for the 'must')

Reply:
Hm, 'it seems' reads to me like he's surprised at the route the train is taking / is only now deducing the route, or that there is somehow a way for the train to take different routes and this time picked the one through Yashirogi. Whereas 'must have' seems more like he knows the route, and by looking at the changed scenery deduced that they have passed through Yashirogi and entered Souya.
Minor niggle though.

Reply:
I'm ok with must have